test/PowerShellEditorServices.Test/Language/LanguageServiceTests.cs

```diff
@@ -289,6 +298,96 @@ public void LanguageServiceFindsSymbolsInNoSymbolsFile()
             Assert.Equal(0, symbolsResult.FoundOccurrences.Count());
         }
 
+        [Theory]
+        [InlineData("3")]
+        [InlineData("4")]
+        [InlineData("5")]
+        public void CompilesWithPowerShellVersion(string version)
+        {
+            var assemblyPath = InstallPackage(string.Format("Microsoft.PowerShell.{0}.ReferenceAssemblies", version), "1.0.0");
```

Comment:
Awesome!  This is a neat way to do it.  Two things I'd recommend:

1. Maybe we can just make the reference assemblies a NuGet dependency of the test project so that we don't need to call NuGet from within the test code?  That way you can just reference the already-pulled dependencies directly.  Might make things easier for AppVeyor

2. It'd be nice to move these tests into their own test class/file, maybe something like PowerShellVersionTests.cs